src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1566,6 +1566,9 @@ protected void Log(string msg, WarningLevel severity)
         /// <param name="workspaceViewInfo">The extra view information from the workspace to update the model with.</param>
         public void UpdateWithExtraWorkspaceViewInfo(ExtraWorkspaceViewInfo workspaceViewInfo)
         {
+            if (workspaceViewInfo == null)
```

Comment:
so if we just return will the view default to something sensible?

Reply:
With no view block the code will skip the section where the view block values are used to modify the loaded workspace. This results in the position and zoom of the view defaulting to 0,0 and 1, all the nodes being crunched together, and no notes or annotations, but this is to be expected with no view information. I can't really say that this is "sensible" per se, but I don't think that there is anything to really be done about it (right now at least).

I will need to do some more testing to determine the specifics, but I have tested with both no view block at all and with a view block and no Dynamo preferences and there are no crashes. In the case of no Dynamo preferences block there are some default values that will be used instead.